README.md

```diff
@@ -49,7 +49,7 @@ To make the signature familiar to readers, we use common JavaScript idioms such
 Optional parameters can be indicated with `?`:
 
 ```js
-User({ name: string, avatarUrl? }): User
+(param: type, optParam?: type): returnType
```

Comment:
I felt it was a little early to use a function `interface` as an example. The `interface` keyword gets defined farther down.

Also, optional parameters should still define a type.